### ISSUE TITLE
Conversion from Studio One dawproject to Reaper

### ISSUE THREAD

**Issue by istrunfa** [git-moss/ProjectConverter#3]
Hi, 
Was trying to do my first conversion between files. The Studio One project was about 60 tracks, with multiple regions, meter and signature changes, and no time-stretch events. It was recorded at 88.2kHz, 24bits. I've stripped out every plugin just wanting to test the bare audio regions export.

I kept having the same error message:
`Could not read or parse the source file:
ZipFile invalid LOC header (bad signature)`

Just to be sure, I tried to open the file using Zip software and had no issue. I tried both in Windows and on Mac OSX. Same results.
Then I removed some tracks from the project, leaving around 40 tracks. The error message changed to:

`For input string: "FF0D0DFF" under radix 16
java.lang.NumberFormatException: For input string: "FF0D0DFF" under radix 16
at java.base/java.lang.NumberFormatException.forInputString(Unknown Source)
at java.base/java.lang.Integer.parseInt(Unknown Source)
at java.base/java.lang.Integer.valueOf(Unknown Source)
at java.base/java.lang.Integer.decode(Unknown Source)
at de.mossgrabers.projectconverter.format.reaper.ReaperDestinationFormat.fromHexColor(ReaperDestinationFormat.java:1315)
at de.mossgrabers.projectconverter.format.reaper.ReaperDestinationFormat.createTrack(ReaperDestinationFormat.java:1142)
at de.mossgrabers.projectconverter.format.reaper.ReaperDestinationFormat.convertTrack(ReaperDestinationFormat.java:606)
at de.mossgrabers.projectconverter.format.reaper.ReaperDestinationFormat.convertTracks(ReaperDestinationFormat.java:388)
at de.mossgrabers.projectconverter.format.reaper.ReaperDestinationFormat.write(ReaperDestinationFormat.java:169)
at de.mossgrabers.projectconverter.core.ConversionTask.call(ConversionTask.java:93)
at de.mossgrabers.projectconverter.core.ConversionTask.call(ConversionTask.java:23)
at javafx.concurrent.Task$TaskCallable.call(Task.java:1426)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)`

After a few tries, I changed the color of the tracks to 7FFFFFFF (the maximum value that would fit in a standard Java integer), and it finally succeeded in processing the file, although it only created the audio regions for the first two tracks. Also, these regions were consolidated in the resulting file, and in the original Studio One song were split into many small regions. The rest of the tracks were created with the right names but were empty.

Hope this help develop this amazing and useful tool you are creating!
Thank you for your work!

**Comment by istrunfa** [comment]
> Can you send me the original export?

I can send you a smaller dawproject file (it is the one I'm using to testing). But since the audio is from an actual working session that is still to be released, I cant post it in public. Give me a way to send it privately to you.

**Comment by git-moss** [comment]
Sadly, you still cannot send a PM here. But you can on KVR or the Reaper forum (both "moss"), just send me a link to your favorite file hoster.

**Comment by istrunfa** [comment]
[Test Dawproject.zip](https://github.com/git-moss/ProjectConverter/files/12833185/Test.Dawproject.zip)
I created I quick demo file that illustrates this issue. You have the Studio One file and the resulting dawproject. The project has 50 tracks, audio regions, and tempo changes and the track color is FF007F.
Trying to convert the project throws the error I described.
Turning the tracks black does process the conversion, but the resulting file doesn't have most of the audio regions present, it doesn't have tempo changes or markers. Hope it helps
